src/test/test_gradientcategorial_ruleset.js

```diff
@@ -0,0 +1,65 @@
+var OncoprintRuleSet = require("../js/oncoprintruleset.js");
+var assert = require("chai").assert;
+var $ = require('jquery');
+
+describe("GradientCategoricalRuleSet", function() {
+
+    var mixParams = {
+        type: 'gradient+categorical',
+        legend_label: "this is a label",
+        value_key: "profile_data",
+        value_range: [1,8],
+        value_stop_points: [1,2,8],
+        colors: [[255,0,0,1],[0,0,0,1],[0,255,0,1]],
+        null_color: 'rgba(224,224,224,1)',
+        category_key: "category"
+    }
+
+    var categoryDatum = {
+        category: ">8",
+        profile_data: 8
+    }
+    
+    var gradientDatumLargest = {
+        category: undefined,
+        profile_data: 8
+    }
+
+    var gradientDatumSmallest = {
+        category: undefined,
+        profile_data: 1
+    }
+
+    var naDatum = {
+        category: undefined,
+        profile_data: undefined,
+        truncation: undefined
+    }
+
+    it("Is created from params", function() {
+        var mixRuleSet = new OncoprintRuleSet(mixParams);
+        assert.equal(mixRuleSet.constructor.name, "GradientCategoricalRuleSet");
```

Comment:
as mentioned in slack, this test fails now. lets update it to check instanceof or prototype equality or whatever works.